JokeRandomizer/JokeRandomizer/Views/FirstScreenViewController.swift

```diff
@@ -40,7 +48,7 @@ class FirstScreen: UIViewController {
     }()
 //MARK: - Refresh Button
     private lazy var refreshButton: UIButton = {
```

Comment:
use a generic button that can be configured based on the requirements. do it in other buttons and labels as well

Reply:
move calling of the functions button to CustomButton